woocommerce/woocommerce-android
# Change existing cancel button to keep searching when a card reader is found add a new cancel button

Closes: #6645


### Description

Adds "keep searching" button to the "Reader found screen"

### Testing instructions

* Start connection to a reader
* Notice that there are 3 buttons now
* Try to cancel the connection, notice that it's been canceled
* Try to keep searching and notice that it is searching for more readers

### Images/gif


https://user-images.githubusercontent.com/4923871/171804783-11f9f523-2989-4888-b555-5592cbacfc69.mp4
